apps/console/src/features/users/components/add-user.tsx

```diff
@@ -169,35 +169,35 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
      */
     const handlePasswordChange = (values: Map<string, FormValue>): void => {
         const password: string = values.get("newPassword").toString();
+
         setPassword(password);
     };
 
-    /**
-     * The following function generate a random password.
-     */
     const generateRandomPassword = (): void => {
```

Comment:
Shall we add the doc comment back.

Reply:
Fixed with f1efba513d6385145463c2f9f6462bf4a21f6b17